prestoadmin/plugin.py

```diff
@@ -47,3 +46,15 @@ def add_jar(local_path, plugin_name, plugin_dir=REMOTE_PLUGIN_DIR):
     """
     _LOGGER.info('deploying jars on %s' % env.host)
     write(local_path, os.path.join(plugin_dir, plugin_name))
+
+def remove_jar(local_path, plugin_name):
```

Comment:
For remove jar, there is no local path.  The command should take the plugin name to remove and optionally the remote path to the plugin directory to allow for non-default configurations.

def remove_jar(plugin_name, plugin_dir=REMOTE_PLUGIN_DIR)